src/main/kotlin/com/terraformation/backend/accelerator/db/ApplicationStore.kt

```diff
@@ -428,7 +428,8 @@ class ApplicationStore(
   private fun updatePrescreenFeedback(applicationId: ApplicationId, feedback: List<String>) {
     val feedbackField =
         if (feedback.isNotEmpty()) {
-          feedback.joinToString(separator = "\n")
+          feedback.joinToString(
```

Comment:
We should probably HTML-escape the feedback strings here just so we don't get bitten if at some point down the road we end up adding a problem message that includes a user-entered string value, like "Project name $name is too short" or something.

Spring has a utility function for this, `HtmlUtils.htmlEscape()`.